_posts/hi/newsletters/2022-10-26-newsletter.md

```diff
@@ -0,0 +1,258 @@
+  लिए प्रभावित होने वाली किसी भी सेवा को प्रोत्साहित करता है।
+
+## उल्लेखनीय कोड और दस्तावेज़ीकरण परिवर्तन
+
+*इस सप्ताह [Bitcoin Core][bitcoin core repo], [Core Lightning][core lightning repo],
+[Eclair][eclair repo], [LDK][ldk repo], [LND][LND Repo] में उल्लेखनीय परिवर्तन।
+[libsecp256k1][libsecp256k1 repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][Rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[BDK][bdk repo], [Bitcoin Improvement Proposals (BIP)][BIPs repo], और
+[Lightning BOLTs][bolts repo]।*
+
+- [Bitcoin Core #23443][] एक नया P2P प्रोटोकॉल संदेश जोड़ता है, `sendtxrcncl` (लेनदेन सामंजस्य भेजें), जो एक नोड को एक सहकर्मी को संकेत देने की
+  अनुमति देता है कि यह [erlay][topic erlay] का समर्थन करता है। यह PR erlay प्रोटोकॉल के पहले भाग को जोड़ता है --- इसे उपयोग करने से पहले अन्य भागो की आवशकता है।
+
+- [Eclair #2463][] और [#2461][eclair #2461] Eclair के [इंटरैक्टिव और Dual फंडिंग प्रोटोकॉल][topic dual funding] के कार्यान्वयन को अपडेट करने के लिए [RBF][topic rbf] में प्रत्येक
+  फंडिंग इनपुट ऑप्ट-इन की आवश्यकता होती है। और पुष्टि भी करें (अर्थात ऐसा आउटपुट खर्च करें जो
+  पहले से ही ब्लॉक चेन में हो)। ये परिवर्तन सुनिश्चित करते हैं कि RBF का उपयोग किया जा सकता है और Eclair
+  उपयोगकर्ता द्वारा योगदान की गई किसी भी फीस का उपयोग उनके किसी भी सहकर्मी के पिछले लेनदेन की
+  पुष्टि करने में मदद के लिए नहीं किया जाएगा।
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="23443,2463,2461,25353" %}
+[bitcoin core 24.0 rc2]: https://bitcoincore.org/bin/bitcoin-core-24.0/
+[ldk 0.0.112]: https://github.com/lightningdevkit/rust-lightning/releases/tag/v0.0.112
+[bcc testing]: https://github.com/bitcoin-core/bitcoin-devwiki/wiki/24.0-Release-Candidate-Testing-Guide
+[news222 rbf]: /hi/newsletters/2022/10/19/#transaction-replacement-option
+[kotliar free option]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-October/021056.html
+[sanders cpfp]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-October/021060.html
+[rubin cpfp]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-October/021059.html
+[riard free option]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-October/021067.html
+[zhao no control]: https://www.erisian.com.au/bitcoin-core-dev/log-2022-10-20.html#l-440
+[towns uncoordinated]: https://www.erisian.com.au/bitcoin-core-dev/log-2022-10-20.html#l-488
+[sanders doubt]: https://www.erisian.com.au/bitcoin-core-dev/log-2022-10-20.html#l-490
+[towns uasf]: https://www.erisian.com.au/bitcoin-core-dev/log-2022-10-20.html#l-492
+[zumsande option]: https://www.erisian.com.au/bitcoin-core-dev/log-2022-10-20.html#l-493
+[coredev xs]: https://diyhpl.us/wiki/transcripts/bitcoin-core-dev-tech/
+[p2p encryption]: https://diyhpl.us/wiki/transcripts/bitcoin-core-dev-tech/2022-10-10-p2p-encryption/
+[news222 bip324]: /en/newsletters/2022/10/19/#bip324-update
```

Comment:
/hi/ version available

Reply:
First, I added the `hi` version of the link here. But for some reason, it was causing the following error.

![image](https://user-images.githubusercontent.com/85434418/201466231-7add9b43-0043-4c70-bf64-1e3d7f42848b.png)

So I made an exception to reverting to the `en` version for this link.

Reply:
@shaavan ping on this

Reply:
@shaavan I pushed a change to this link.